### ISSUE TITLE
Child can't be rescued

### ISSUE THREAD

**Issue by papplegate** [mprat/Terminus#88]
I love this game!  I would really like to use Terminus as a teaching tool in the classroom.  However, I think one key puzzle is broken.  

I am reasonably sure that the solution to the frightened child puzzle in the CaveOfDisgruntledTrolls is to use mv like so: 

`mv Cage/KidnappedChild .`

However, this command doesn't seem to work.  The child also isn't recognized by the `less` command.  See the example below.  

`>cd CaveOfDisgruntledTrolls
You have moved to CaveOfDisgruntledTrolls. The cave is dark and smells like... feet? 
Oh, right, it's probably the trolls. There's a scared-looking kid in a cage by the 
far wall. 
>ls
 Locations: 
Cage
Slide
 Items: 
UglyTroll
UglierTroll
AbsolutelyHideousTroll
>mv Cage/KidnappedChild ../KidnappedChild
Must be a valid item and location to move it.
>mv Cage/KidnappedChild ./KidnappedChild
Must be a valid item and location to move it.
>less Cage/KidnappedChild
There is no undefined here.
>less UglierTroll
He looks mad, and really, really ugly. But he wants to tell you something. Between 
his garbled grunts, you manage to understand the following: You can cast spells on 
items that you don't currently see. For example, if you want to copy an item from 
this room to the OminousLookingPath from which you came, you can cp [ITEM_TO_COPY] 
../[NEW_ITEM_NAME]. You can do this for most spells and most rooms. Use this 
knowledge wisely. Remember.... you can't undo an 'rm' spell. 
>ls Cage
 Locations: 

 Items: 
KidnappedChild
>less Cage/KidnappedChild
There is no undefined here.`

**Comment by Nakpangi** [comment]
I know that I'm about two years too late here, but the solution is that you've gotta move the UglyTroll into the cage. Unfortunately, we care more about getting to the slide than we do about the kidnapped child's well being.

The command/magic spell is:

`mv UglyTroll Cage`

**Comment by fayefossegrim** [comment]
> I know that I'm about two years too late here, but the solution is that you've gotta move the UglyTroll into the cage. Unfortunately, we care more about getting to the slide than we do about the kidnapped child's well being.
> 
> The command/magic spell is:
> 
> `mv UglyTroll Cage`

yes but ONLY after u defeat the trolls ,  or smth cuz they.ll eat the kid in the cage

**Comment by kool09** [comment]
> > I know that I'm about two years too late here, but the solution is that you've gotta move the UglyTroll into the cage. Unfortunately, we care more about getting to the slide than we do about the kidnapped child's well being.
> > The command/magic spell is:
> > `mv UglyTroll Cage`
> 
> yes but ONLY after u defeat the trolls , or smth cuz they.ll eat the kid in the cage

It didn't work despite me doing both of these. There's no way to access child items of any room at all (yeah i'm like 5 years late to the whole debacle)

**Comment by AaronBeck808** [comment]
https://web.mit.edu/mprat/Public/web/Terminus/Java/CMS.590Game2FinalReport.pdf

Found this Gem of what should've happened if quest were to be completed
